pkg/tracker/index.go

```diff
@@ -84,12 +84,17 @@ func parseIndexFile(cfg *config.Config, DB db.DB) (trackersPerURL map[string]*In
 					source = &JSONfile{filepath: filepath.Join(cfg.ConfigFolder, api.URL)}
 					name = filepath.Base(api.URL)
 				}
+				minimumInterval := uint(cfg.TrackerSleepCycle.Duration) / 2
```

Comment:
Why is this divided by 2?

Reply:
I was debating this...say the trackerCycle is 10 seconds.  If we check 9 seconds later, should it update if no interval specified?

Reply:
yeah, I'm deleting it, I think keep it flat and query these things less

Reply:
But we should deff have some check that the custom API interval is the same or smaller than the global  interval